config/routes.rb

```diff
@@ -113,6 +113,7 @@
     end
     resource :identify_types_of_income, only: %i[show update]
     resource :cash_income, only: %i[show update]
+    resource :cash_outgoing, onlu: %i[show update]
```

Comment:
typo here - should be `only` I assume. I have no idea how this is still working correctly.